public/app/features/datasources/settings/DataSourceSettingsPage.tsx

```diff
@@ -140,6 +142,14 @@ export class DataSourceSettingsPage extends PureComponent<Props> {
     );
   }
 
+  renderMissingEditRightsMessage() {
+    return (
+      <Alert aria-label={selectors.pages.DataSource.readOnly} severity="info" title="Missing rights">
```

Comment:
Is the `aria-label` used for tests? Maybe switch to use data-testid instead. Please refer to this doc: https://github.com/grafana/grafana/blob/main/contribute/style-guides/e2e.md#aria-labels-vs-data-testid